sailor/assetcentral/system.py

```diff
@@ -5,26 +5,27 @@
 """
 from __future__ import annotations
 
+import math
 import itertools
 import logging
-from typing import TYPE_CHECKING, Union
+from typing import Union
 from datetime import datetime
 from functools import cached_property
 from operator import itemgetter
 
 import pandas as pd
 
-from sailor import _base
-from sailor import sap_iot
+from sailor import _base, sap_iot
+import sailor.assetcentral.indicators as ac_indicators
 from sailor.utils.utils import WarningAdapter
 from .utils import (AssetcentralEntity, _AssetcentralField, AssetcentralEntitySet,
                     _ac_application_url, _ac_fetch_data)
 from .equipment import find_equipment, EquipmentSet
-from .indicators import IndicatorSet
+from .indicators import (IndicatorSet, SystemIndicator, SystemAggregatedIndicator, SystemIndicatorSet,
+                         SystemAggregatedIndicatorSet)
+from ..sap_iot import TimeseriesDataset
```

Comment:
please remove the import of TimeseriesDataset.

As `sap_iot` is already imported, use it down below with `sap_iot.TimeseriesDataset`